tests/Operation/CreateEncryptedCollectionFunctionalTest.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace MongoDB\Tests\Operation;
+
+use MongoDB\BSON\Binary;
+use MongoDB\Client;
+use MongoDB\ClientEncryption;
+use MongoDB\Driver\WriteConcern;
+use MongoDB\Operation\CreateEncryptedCollection;
+
+use function base64_decode;
+use function explode;
+use function getenv;
+use function is_executable;
+use function is_readable;
+use function version_compare;
+
+use const DIRECTORY_SEPARATOR;
+use const PATH_SEPARATOR;
+
+class CreateEncryptedCollectionFunctionalTest extends FunctionalTestCase
+{
+    public const LOCAL_MASTERKEY = 'Mng0NCt4ZHVUYUJCa1kxNkVyNUR1QURhZ2h2UzR2d2RrZzh0cFBwM3R6NmdWMDFBMUN3YkQ5aXRRMkhGRGdQV09wOGVNYUMxT2k3NjZKelhaQmRCZGJkTXVyZG9uSjFk';
+
+    /** @var ClientEncryption */
+    private $clientEncryption;
+
+    public function setUp(): void
```

Comment:
Most of this is lifted from the CSFLE prose tests. Copying this over reminded me of our previous discussion about using traits (vs. piling more onto the base FunctionalTestCase class), but that's something for a future PR.